source/common/resp_cache/simple_cache.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
This file needs to be under `api/`. This is a new type of extension. It could either go in `api/envoy/extensions/ext_authz_cache/simple`, or maybe it belongs under `api/envoy/extensions/filters/http/ext_authz/cache/simple`. @envoyproxy/api-shepherds can you provide guidance on the correct location?